### PR TITLE
#161568013 add jwt to endpoints

### PR DESCRIPTION
**Pull request by MbugwaSami**
**What does this PR do?**
_This PR adds chores to current running endpoints _

**Description of Task to be completed:**
_Add JWT tokens to user endpoints_
_Add JWT tokens to products endpoints_

**How to manually test**
- git clone https://github.com/MbugwaSami/ADC3-Store-Manager-Api.git
- cd ADC3-Store-Manager-Api
- pip install virtualenv
- source venv/scripts/activate
- git checkout  ch-setup-application-#161469685
- python run.py to run the app

**Relevant PT story**
#161568013

### CODE REVIEW

File: app/api/v2/views/users_endpoint.py [hunk 1]
```diff
@@ -27,28 +35,26 @@ def post(self):
...
         user_details = [email,names,password,role]
-        print(user_object.get_all_users())
         for field in user_details:
             if not field or field.isspace():
-                return {'message':'Some fields are empty!'}
+                return jsonify({'message':'Some fields are empty!'})
 
         if not user_object.validate_email(email):
-            return {'message':'Please enter a valaid email'}
+            return jsonify({'message':'Please enter a valaid email'})
```
Comment (dotNesh) [inline app/api/v2/views/users_endpoint.py]: Correct the spelling mistake. Its valid not valaid.

File: app/api/v2/views/users_endpoint.py [hunk 2]
```diff
@@ -78,8 +84,11 @@ def post(self):
         password = data.get('password')
 
         if not user_object.verify_user(email,password):
-            return {'message':'wrong email or password'}
+            return jsonify(dict(message = "wrong email or password"))
```
Comment (dotNesh) [inline app/api/v2/views/users_endpoint.py]: Use a dictionary eg. {"message": "wrong email or password"}

**Review by dotNesh — COMMENTED** [review]
Remove the code on the __init__ file in the test folder and put it in a test_base.py

**Comment by dotNesh** [comment]
Work on the requested changes, especially how you return the responses. Ensure to have status codes for each response.

File: app/api/v2/views/users_endpoint.py (hunk 1 above)
Comment (MbugwaSami) [reply app/api/v2/views/users_endpoint.py]: done

File: app/api/v2/views/users_endpoint.py (hunk 2 above)
Comment (MbugwaSami) [reply app/api/v2/views/users_endpoint.py]: done

**Pull request merged**